index.js

```diff
@@ -22,7 +22,7 @@ module.exports = function(options) {
       paths.push(path);
     }
 
-    if (file.isBuffer() && path === 'index.html') {
+    if (file.isBuffer() && path === entryPoint) {
```

Comment:
@gmarty dont we need to fix the examples, or at least provide a default value for `entryPoint`?